topi/python/topi/cuda/conv2d_winograd.py

```diff
@@ -375,6 +375,12 @@ def _alter_conv2d_layout(attrs, inputs, tinfos):
         if cfg.template_key == 'direct':
             return None
 
+        if cfg.template_key == 'int8':
+            new_attrs['layout'] = 'NCHW4c'
```

Comment:
_alter_conv2d_layout is called from opencl and rocm backend too. They don't support int8, so it is better to add an assertion here.
 
```
assert tvm.target.current_target() in ["cuda", "nvptx"]
``` 

I'm not sure if dp4 intrinsic works on nvptx backend. If not, you can remove "nvptx" from above.